savoirfairelinux/sous-chef
# Fixed #672: Added placeholder to member's field in client form

## Fixes # aaboffill

### Changes proposed in this pull request:

* Add a placeholder more descriptive ("Type more than 2 characters to search Members") for the member's fields inside the `Client`'s form, with the purpose of make more intuitive those fields.

![image](https://cloud.githubusercontent.com/assets/2370639/24149370/b06421ec-0e18-11e7-9ff9-f36789c5d657.png)

### Status


### How to verify this change

* Go to create or update some `Client`, check the member fields into the tabs "Referent", "Payment" or "Emergency".

### New translatable strings
